core/sequencer/server.go

```diff
@@ -529,12 +532,14 @@ func (s *Server) HighWatermarks(ctx context.Context, directoryID string, lastMet
 	// revision.
 	// TODO(gbelvin): Separate end watermarks for the sequencer's needs
 	// from ranges of watermarks for the verifier's needs.
-	ends := make(map[int64]int64)
-	starts := make(map[int64]int64)
+	ends := make(map[int64]time.Time)
+	starts := make(map[int64]time.Time)
```

Comment:
nit: maybe one for a different PR, but defining `ends` before `starts` (and assigning in that order below) makes my inner narrator twitch.